pkg/chart/dependency.go

```diff
@@ -88,9 +100,39 @@ func syncDependencies(chartPath string, sourceRepo *api.Repo, target *api.Target
 	return errs
 }
 
+// updateChartMetadataFile updates the dependencies in Chart.yaml
+// For helm v3 dependency management
+func updateChartMetadataFile(chartPath string, lock *helmChart.Lock, sourceRepo *api.Repo, target *api.TargetRepo) error {
+	chartFile := path.Join(chartPath, ChartFilename)
+	chart, err := ioutil.ReadFile(chartFile)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	chartMetadata := &helmChart.Metadata{}
+	err = yaml.Unmarshal(chart, chartMetadata)
+	if err != nil {
+		return errors.Annotatef(err, "Error unmarshaling %s file", chartFile)
```

Comment:
remember that errors should not be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Just for you to know, the new errors I wrote in this PR I did it in lower case, but this one comes from a copy-paste.

Hopefully, you don't have to remind me this rule anymore 😄